admin_interface/settings.py

```diff
@@ -23,6 +23,5 @@ def check_installed_app(app, app_dj_version_limit):
 
 
 def check_installed_apps():
-    check_installed_app('colorfield', (4, 0))
```

Comment:
Any reason not to make this check conditional on version?
Django 4.0 isn’t out yet, and it is common to stay on LTS versions, so checking on 3.2 would still be useful.
https://www.djangoproject.com/download/#supported-versions

Reply:
This check is not useful at all, with it the library will surely raise an Exception even if not needed (like in this case).

Reply:
Agree and for 3.2 it is not relevant (with or without)